integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -0,0 +1,140 @@
+public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/vault"
```

Comment:
We should remove the public_url now. It returns a 404 as is. It will be set by the post release job that will be run.

https://github.com/GoogleCloudPlatform/ops-agent/pull/632#discussion_r905486897

Reply:
Updated